presto-product-tests/bin/run_on_docker.sh

```diff
@@ -81,8 +81,11 @@ function stop_application_runner_containers() {
     echo "Container stopped: ${CONTAINER_NAME}"
   done
   echo "Removing dead application-runner containers"
-  docker ps -aq --no-trunc --filter status=dead --filter status=exited --filter name=common_application-runner \
-  | xargs docker rm -v || true
```

Comment:
This was the first thing i tried. But Mac version of `xargs` doesn't have such flag unfortunately.